bitmap-ittybitty/README.md

```diff
@@ -0,0 +1,8 @@
+#Itty Bitty Bitmap Transformer
```

Comment:
Nice README! People always forget these, but they're one of the most important files to have.